### DISCUSSION TITLE
[QUESTION] SideMenuView - Collapse Left or Right Menu

### DISCUSSION THREAD

**Discussion by angelru** [xamarin/XamarinCommunityToolkit#1073] [Q&A] [answered]
Let's say that in any of the side menus I have options, how do I collapse the menu when I click on any of them?

**Comment by AndreiMisiukevich** [comment]
Hi @angelru 
SideMenuView has a property "State". Just set desired state and the view will start the animation.

**Reply by angelru** [reply]
but how do I change the state from the ViewModel?

**Reply by acaliaro** [reply]
Use a IValueConverter or a DataTrigger

**Comment by angelru** [comment, marked as answer]
Thanks!
https://github.com/xamarin/XamarinCommunityToolkit/issues/924
